notebook/notebookapp.py

```diff
@@ -645,6 +646,9 @@ def nbextensions_path(self):
             path.append(os.path.join(get_ipython_dir(), 'nbextensions'))
         return path
 
+    nbextensions = List(Unicode(), config=True,
+        help="A list of nbextensions to enable.")
+
```

Comment:
frontend-loaded extensions won't be server-side config that requires restart, will they?